Description: Fix hyperlinks from the C++ library documentation to python-vigra documentation and the other way round.
Bug-Debian: http://bugs.debian.org/590069
Bug-Debian: http://bugs.debian.org/590068
Forwarded: not-needed
Last-Update: 2011-03-02

Index: b/docsrc/credits_changelog.dxx
===================================================================
--- a/docsrc/credits_changelog.dxx
+++ b/docsrc/credits_changelog.dxx
@@ -42,7 +42,7 @@
       <li> Michael Hanselmann and Martin Lindner implemented much of the HDF5 support.
       
       <li> Nathan Huesken and Christoph Sommer helped a lot in developing 
-           <a href="../vigranumpy/index.html">vigranumpy</a>, the VIGRA Python bindings.
+           <a href="/usr/share/doc/python-vigra-doc/html/index.html">vigranumpy</a>, the VIGRA Python bindings.
            
       <li> Jens-Malte Gottfried contributed to the cmake-based build system.
 
@@ -154,7 +154,7 @@
       
       <li> Added \ref vigra::NumpyAnyArray and \ref vigra::NumpyArray, improved \ref vigra::MultiArrayView.
       
-      <li> Added <b><a href="../vigranumpy/index.html">vigranumpy</a></b> (VIGRA Python bindings).
+      <li> Added <b><a href="/usr/share/doc/python-vigra-doc/html/index.html">vigranumpy</a></b> (VIGRA Python bindings).
       
       <li> Added \ref VigraMatlab.
       
Index: b/docsrc/index.dxx
===================================================================
--- a/docsrc/index.dxx
+++ b/docsrc/index.dxx
@@ -32,7 +32,7 @@
         <BR>&nbsp;&nbsp;&nbsp;<em>Demonstration programs for VIGRA's usage </em>
     <LI> \ref VigraMatlab
         <BR>&nbsp;&nbsp;&nbsp;<em>VIGRA Matlab bindings</em>
-    <LI> <b><a href="../vigranumpy/index.html">vigranumpy</a></b>
+    <LI> <b><a href="/usr/share/doc/python-vigra-doc/html/index.html">vigranumpy</a></b>
         <BR>&nbsp;&nbsp;&nbsp;<em>VIGRA Python bindings</em>
     <LI> \ref CreditsChangelog
         <BR>&nbsp;&nbsp;&nbsp;<em>Who contributed what?</em>
Index: b/vigranumpy/docsrc/conf.py.in
===================================================================
--- a/vigranumpy/docsrc/conf.py.in
+++ b/vigranumpy/docsrc/conf.py.in
@@ -239,6 +239,6 @@
     .. |%s| replace:: %s
 """% (replacement[0],replacement[0][k+1:])
     rst_epilog += """
-    .. _%s: ../../doc/vigra/%s
+    .. _%s: /usr/share/doc/libvigraimpex-doc/html/%s
 """% (replacement[0],replacement[1])
 
Index: b/vigranumpy/docsrc/index.rst
===================================================================
--- a/vigranumpy/docsrc/index.rst
+++ b/vigranumpy/docsrc/index.rst
@@ -14,7 +14,7 @@
 Introduction
 ------------
 
-Vigranumpy exports the functionality of the C++ image processing library `VIGRA <../vigra/index.html>`_ to Python. It can be invoked by importing the vigra module::
+Vigranumpy exports the functionality of the C++ image processing library `VIGRA </usr/share/doc/libvigraimpex-doc/html/index.html>`_ to Python. It can be invoked by importing the vigra module::
 
     import vigra
 
@@ -260,7 +260,7 @@
 
 The module vigra.colors provides functions to adjust image brightness and contrast, 
 and to transform between different color spaces. 
-See `Color Conversions <../vigra/group__ColorConversions.html>`_ in the C++ documentation
+See `Color Conversions </usr/share/doc/libvigraimpex-doc/group__ColorConversions.html>`_ in the C++ documentation
 for more information.
 
 .. automodule:: vigra.colors
